ui/pages/settings/experimental-tab/experimental-tab.component.js

```diff
@@ -89,24 +102,47 @@ export default class ExperimentalTab extends PureComponent {
           className="settings-page__content-row settings-page__content-row-experimental"
         >
           <div className="settings-page__content-item">
-            <span>{t('securityAlerts')}</span>
+            <Text
+              variant={TextVariant.bodyMd}
+              color={TextColor.textAlternative}
+            >
+              {t('securityAlerts')}
+            </Text>
             <div className="settings-page__content-description">
+              <Text variant={TextVariant.bodySm}>
+                {t('securityAlertsDescription')}
+              </Text>
               <Text
-                variant={TextVariant.bodySm}
+                variant={TextVariant.bodySmBold}
                 color={TextColor.textAlternative}
+                marginTop={4}
               >
-                {t('securityAlertsDescription')}
+                {t('preferredProvider')}
               </Text>
               <div
                 data-testid="settings-toggle-security-alert-blockaid"
                 className="settings-page__content-item-col settings-page__content-item-col__security-toggle-option"
               >
-                <Text
-                  variant={TextVariant.bodyMd}
-                  color={TextColor.textDefault}
-                >
-                  {t('blockaid')}
-                </Text>
+                <div>
+                  <Box display={Display.Flex}>
+                    <Text
+                      variant={TextVariant.bodyMd}
+                      color={TextColor.textDefault}
+                    >
+                      {t('blockaid')}
+                    </Text>
+                    <Tag marginLeft={2} label="Recommended" />
+                  </Box>
+                  <Text
+                    variant={TextVariant.bodySm}
+                    as="h6"
+                    color={TextColor.textAlternative}
+                    marginTop={0}
+                    marginRight={1}
```

Comment:
thinking we will want to use margin-inline-end or padding-inline-end here and the other elements below as well